spec/identity_doc_auth/acuant/responses/get_results_response_spec.rb

```diff
@@ -187,5 +187,283 @@
         expect(response.exception).to be_nil
       end
     end
+
+    context 'when front image HDPI is too low' do
+      let(:http_response) do
+        parsed_response_body = JSON.parse(AcuantFixtures.get_results_response_failure)
```

Comment:
The fixture is parsed in each of these contexts. You could wrap these in a context and pull that out, e.g.,
```
    context 'image metrics errors' do
      let(:parsed_response_body) { JSON.parse(AcuantFixtures.get_results_response_failure) }

      context 'when front image HDPI is too low' do
        let(:http_response) do
          parsed_response_body['Images'].first['HorizontalResolution'] = 250
          instance_double(
            Faraday::Response,
            body: parsed_response_body.to_json,
          )
        end
...
```

Reply:
good call. Don't need to parse each time.